src/pages/search.tsx

```diff
@@ -23,13 +27,71 @@ const Search = () => {
   const playlists = usePlaylistSearch(searchString)
   const artists = useArtistSearch(searchString)
 
+  const hasMusics = !!musics.data?.length
+  const hasPlaylists = !!playlists.data?.length
+  const hasArtists = !!artists.data?.length
+
   console.log('musics', musics.data)
   console.log('playlists', playlists.data)
   console.log('artists', artists.data)
 
   return (
-    <Stack>
+    <Stack spacing={3.5} sx={{ margin: '40px 5%' }}>
```

Comment:
what is 5% of margin?

Reply:
due to the figma margin being too large and not responsive. I change it to how I wish it to be